gravitee-apim-console-webui/src/management/management.module.ajs.ts

```diff
@@ -706,7 +706,7 @@ graviteeManagementModule.component('categories', CategoriesComponent);
 graviteeManagementModule.component('category', CategoryComponent);
 graviteeManagementModule.component('moved', MovedComponent);
 
-graviteeManagementModule.component('metadata', MetadataComponent);
+graviteeManagementModule.directive('metadata', downgradeComponent({ component: EnvironmentMetadataComponent }));
```

Comment:
To make it easier to distinguish between what is migrated and what is not, we name the components as follows
(`ng` prefix)
```suggestion
graviteeManagementModule.directive('ngEnvrionementMetadata', downgradeComponent({ component: EnvironmentMetadataComponent }));
```

Reply:
Done